pagopa/pn-frontend
# fix duplicated payment

## Short description

fix bug when getPaymentNotifcatioUrl was wrong
## List of changes proposed in this pull request
- delete in redux add same array in same array 
- 
## How to test
go in dev enviroment, with ada user any iun with payment. With netify extension cause the call to fail 
https://webapi.dev.notifichedigitali.it/bff/v1/payments/cart and check if payment card not duplicate
